frontend/src/components/UserInfo/UserAvatarWithInfo.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Redirect } from 'react-router';
+import {withStyles} from '@material-ui/core/styles';
+import UserInfo from "./UserInfo";
+import UserAvatar from "./UserAvatar";
+
+const styles = {
+    forDiv: {
+        flexGrow: 1,
```

Comment:
Don't need `flex-grow` property here. By default flex container stretch to fill the space

Reply:
Ok. Thanks!